README.md

```diff
@@ -1,5 +1,10 @@
 Open-World
-===========
+### Requirments
+- Install Node.js
+- npm install socket.io
+- Homebrew 
+- xCode 
```

Comment:
I don't think `Homebrew` and `Xcode` and required. These are both macOS apps, too, and since this is a Node.js and web-app there should be no need for them.

If you are using Homebrew to install Node.js -- which I do -- that is up to you, the "user" to choose your Node.js installation path. Not dictated by this project.

Reply:
Okay, will change it.